### DISCUSSION TITLE
The collections cannot be physically deleted

### DISCUSSION THREAD

**Discussion by register2hard** [outline/outline#7338] [Ideas]
When clearing the recycle bin, is it possible to physically delete the collections that has been marked as deleted?

**Comment by register2hard** [comment]
and can be cleanup by cron task

**Reply by tommoor** [reply]
Hi, deleted collections currently do not appear in the trash – once they are removed from the sidebar they cannot be restored.

**Reply by register2hard** [reply]
Thank you for your reply! 
After the collection was deleted, it still exists in the database and is marked as deleted using deletedAt. Since it cannot be restored, can it be actually deleted from the database？

**Reply by tommoor** [reply]
It's designed to remain, so you can try but I'm not sure what the knock on effects might be with eg document relations

**Reply by register2hard** [reply]
Ok, I know. Thanks a lot.
